src/Core/src/Handlers/Button/ButtonHandler.iOS.cs

```diff
@@ -182,16 +183,24 @@ public void Disconnect(UIButton platformView)
 				platformView.TouchUpInside -= OnButtonTouchUpInside;
 				platformView.TouchUpOutside -= OnButtonTouchUpOutside;
 				platformView.TouchDown -= OnButtonTouchDown;
+				platformView.TouchCancel -= OnButtonTouchCancel;
 			}
 
-			void OnButtonTouchUpInside(object? sender, EventArgs e)
-			{
-				if (VirtualView is IButton virtualView)
-				{
-					virtualView.Released();
-					virtualView.Clicked();
-				}
-			}
+			void HandleButtonInteraction()
+            {
+                VirtualView?.Released();
+                VirtualView?.Clicked();
```

Comment:
Do we still want to trigger clicked if this is called from `TouchCancel` ?

Reply:
As per the suggestion, we cross-checked and confirmed that the visual state of the button is reset without invoking the clicked event during `TouchCancel`. Could you please verify this?